tests/python/unittest/test_meta_schedule_tune_tir.py

```diff
@@ -66,6 +66,7 @@ def test_tune_matmul_cpu():
                 max_trials_global=32,
             ),
             work_dir=work_dir,
+            blocks=["update"],
```

Comment:
Can you please add another `tune_block` test case instead to check if `tune_tir` can work for the case where block is specified? Specifically you can create a workload with 2 blocks and apply `ComputeInline` to one of the blocks. It should work if the api works as expected.

Reply:
I attempted to add a new test that covers this. Can you confirm it's what you were thinking?

Reply:
Thanks for adding the test and it works in this case. Furthermore, we are assuming there's an AutoInline schedule rule which would inline block 'A' if available so that we can tell if `tune_block` api works. We could simplify and avoid this assumption by directly supplying it with a schedule rule to inline the block it gets so that it will trigger an error if blocks api is not correctly called (because we cannot inline both blocks right).

Reply:
Here is an example that you may follow:
```
@pytest.mark.skip("Integration test")
def test_tune_block_cpu():
    @derived_object
    class RemoveBlock(PyScheduleRule):
        def _initialize_with_tune_context(self, context: TuneContext) -> None:
            pass

        def apply(self, sch: Schedule, block: BlockRV) -> List[Schedule]:
            if sch.get(block).name_hint == "root":
                return [sch]
            sch = sch.copy()
            sch.compute_inline(block)
            return [sch]

    with tempfile.TemporaryDirectory() as work_dir:
        sch: Schedule = tune_tir(
            mod=two_step,
            target=Target("llvm --num-cores=16"),
            config=TuneConfig(
                strategy="replay_trace",
                num_trials_per_iter=32,
                max_trials_per_task=32,
                max_trials_global=32,
            ),
            work_dir=work_dir,
            blocks=["A"],
            sch_rules=lambda *args: [RemoveBlock()],
        )
        assert sch is not None
```

Reply:
That makes sense. Thanks Xiyou!